src/app/components/list/list.service.js

```diff
@@ -41,16 +37,24 @@
 
     // Cache a list for future offline use
     List.prototype.cache = function () {
+      var lfusers = $localForage.instance('users');
+      var lflists = $localForage.instance('lists');
       var request = { limit: 50, offset: 0, sort: 'name'};
+      lflists.setItem(this._id, this);
```

Comment:
I don't really think that's needed. The way offline caching works now is that it caches only the lists you're part of and your favorite lists, and when you go offline, it uses the indexeddb databases instead of pulling from the API, so if a list or user can not be set, when you go offline, it simply won't show up...